src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -31,6 +31,12 @@
             <argument type="service" id="translator.selector" />
         </service>
 
+        <service id="translator.formatter" alias="translator.formatter.trans_message" />
+
+        <service id="translator.formatter.trans_message" class="Symfony\Component\Translation\Formatter\TransMessageFormatter" public="false">
```

Comment:
The framework bundle will now need the [translation component from `~3.2`](https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/composer.json#L36) to make tests green.